addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -24,6 +24,18 @@
     transform: scale(-1);
 }
 
+// Needed to prevent the emoji picker from inheriting theme colors
+.o-EmojiPicker {
+    --o-EmojiPicker-bg: white;
+    --o-EmojiPicker__section-bg: white;
+    --o-EmojiPicker-color: black;
+    --o-EmojiPicker__input-color: black;
+    --o-EmojiPicker__input-placeholder-color:  #{rgba(black, $o-opacity-muted)};
+    --o-EmojiPicker__input-font-size: 1rem;
+    --o-EmojiPicker--active: #e7e9ed;
+    --o-EmojiPicker__navbar--active: #F9FAFB;
+}
```

Comment:
Thanks @chgo-odoo 👍 
Looks like this block of code it's not triggered in the frontend. Could you double-check? Thanks!

<img width="1219" alt="image" src="https://github.com/user-attachments/assets/e6af0619-1d91-4626-ace7-a42d5976d9ef">

Reply:
Indeed @stefanorigano, the issue is mainly in eLearning. As AKU mentioned on the task, they managed to [wrap some components within a shadow DOM](https://github.com/odoo/odoo/blob/17.0/addons/im_livechat/static/src/embed/common/boot_helpers.js#L55), but it's not something affecting the whole front-end atm. This PR provides a global approach and makes use of CSS variables to handle both environment.